wimaging/wimaging_provision_unattend.xml.erb

```diff
@@ -13,9 +13,15 @@ params:
 - windowsLicenseOwner: Company, INC # Legal owner of the Windows license key
 - systemLocale: en-US
 - systemUILanguage: en-US
-- systemTimeZone: Pacific Standard Time #see https://msdn.microsoft.com/en-us/library/ms912391(v=winembedded.11).aspx
+- systemTimeZone: GMT Standard Time #see https://msdn.microsoft.com/en-us/library/ms912391(v=winembedded.11).aspx
```

Comment:
Sounds like UTC would make more sense, as it is a world standard for universal time.